packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md

```diff
@@ -273,13 +273,13 @@ The remaining authenticator should display the phone factor to the user. Since t
 
 The user can either enroll in the phone factor or skip the phone factor. Your code should handle both scenarios that will be described in the following steps.
 
-### Step 18a, Option 1, Enroll and verify the phone authenticator
+### 18a, Option 1, Select phone authenticator
```

Comment:
In this one you've got the structure `number, option number, step heading`, and in the one below you've got the structure `number: option number: step heading`. This needs to be consistent, and I think we should do `number, option number: step heading` — keep the number and option the same as it is written in the diagrams, but keep the meta data and step heading separated by a colon.